### PR TITLE
Cron Support

### PR DESCRIPTION
**Pull request by ArmaanT**
This PR adds cron support. It's a WIP and hasn't been tested as of yet.

### CODE REVIEW

File: templates/_helpers.tpl [hunk 1]
```diff
@@ -19,4 +19,22 @@
 {{/* persist list of apps */}}
 {{- $_ := set . "apps" $apps }}
 
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{- define "setCronJobs" }}
```
Comment (pawalt) [inline templates/_helpers.tpl]: why not bundle this into setdefaults?

File: templates/crontabs.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  schedule: {{ .schedule | quote }}
+  successfulJobsHistoryLimit: {{ .successLimit }}
+  failedJobsHistoryLimit: {{ .failureLimit }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: {{ .restartPolicy }}
+          {{- if .serviecAccountName -}}
+          serviceAccountName: {{ .serviecAccountName }}
```
Comment (pawalt) [inline templates/crontabs.yaml]: typo

**Review by pawalt — CHANGES_REQUESTED** [review]
please also make an example cronjob spec and test it with:
```bash
helm template --name-template cronjob -f example_values.yaml . | kubectl apply --dry-run -f -
```

File: templates/_helpers.tpl (hunk 1 above)
Comment (ArmaanT) [reply templates/_helpers.tpl]: Good point. Changed.

File: templates/crontabs.yaml (hunk 2 above)
Comment (ArmaanT) [reply templates/crontabs.yaml]: Fixed.

**Review by pawalt — APPROVED** [review]
This looks great! Thanks for adding some fuzz to our tests. Let's work on putting tests in CI soon.

**Pull request merged**